projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -66,43 +63,11 @@ export class CartItemListComponent implements OnInit, OnDestroy {
     }
   }
 
-  /**
-   * @deprecated since version 3.1
-   * Use constructor(activeCartService: ActiveCartService, selectiveCartService: SelectiveCartService, featureConfigService: FeatureConfigService, userIdService: UserIdService, multiCartService: MultiCartService); instead
-   */
-  // TODO(#11037): Remove deprecated constructors
-  constructor(
-    activeCartService: ActiveCartService,
-    selectiveCartService: SelectiveCartService
-  );
-
-  /**
-   * @deprecated since version 3.2
-   * Use constructor(activeCartService: ActiveCartService, selectiveCartService: SelectiveCartService, featureConfigService: FeatureConfigService, userIdService: UserIdService, multiCartService: MultiCartService); instead
-   */
-  // TODO(#11037): Remove deprecated constructors
-  constructor(
-    activeCartService: ActiveCartService,
-    selectiveCartService: SelectiveCartService,
-    // eslint-disable-next-line @typescript-eslint/unified-signatures
-    featureConfigService: FeatureConfigService
-  );
-
-  constructor(
-    activeCartService: ActiveCartService,
-    selectiveCartService: SelectiveCartService,
-    featureConfigService: FeatureConfigService,
-    userIdService: UserIdService,
-    // eslint-disable-next-line @typescript-eslint/unified-signatures
-    multiCartService: MultiCartService
-  );
-
   constructor(
     protected activeCartService: ActiveCartService,
     protected selectiveCartService: SelectiveCartService,
-    public featureConfigService?: FeatureConfigService,
-    protected userIdService?: UserIdService,
-    protected multiCartService?: MultiCartService
+    protected userIdService: UserIdService,
+    protected multiCartService: MultiCartService
   ) {}
```

Comment:
Add migration to changed constructor and mention breaking changes in constructor in 4_0.md doc